pages/common/hut.md

```diff
@@ -0,0 +1,29 @@
+# hut
+
+> A CLI tool for sourcehut.
+> Use `hg` in place of `git` below if using mercurial.
+> See `man hut` for more details.
+
+- Initialize hut's configuration file:
+
+`hut init`
+
+- List git repositories:
+
+`hut git list`
+
+- Create a git repository:
+
+`hut git create`
+
+- List jobs on builds.sr.ht:
```

Comment:
```suggestion
- List jobs on <builds.sr.ht>:
```
We wrap links inside angular brackets (as suggested in the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links)).